packages/flutter/lib/src/material/date_picker.dart

```diff
@@ -581,14 +597,24 @@ class _DatePickerDialogState extends State<DatePickerDialog> with RestorationMix
           : localizations.datePickerHelpText.toUpperCase()
```

Comment:
Use the new `useMaterial3` variable here instead.

Reply:
Addressed in https://github.com/flutter/flutter/pull/119033/commits/70d860d720f506a57c730749eaaba252e4cd8a6b